packages/components/src/LightBox/LightBox.mdx

```diff
@@ -0,0 +1,95 @@
+---
+name: LightBox
+menu: Components
+route: /components/light-box
+---
+
+import { useState } from "react";
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Button } from "../Button";
+import { LightBox } from ".";
+
+# Light Box
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+LightBox is a component designed to display an array of images to the user. A
+user clicks on a control and is shown a carousel style presentation of the
+defined images.
+
+This component is a wrapper of the frontend-collective's react-image-lightbox
+third party library.
+<a href="https://github.com/frontend-collective/react-image-lightbox">React
+Image Lightbox</a>
+
+```ts
+import { LightBox } from "@jobber/components/LightBox";
+```
+
+<Playground>
+  {() => {
+    const [open, isOpen] = useState(false);
+    return (
+      <>
+        <Button label="😈 Click me!" onClick={() => isOpen(true)} />
+        <LightBox
+          open={open}
+          images={[
+            {
+              title: "😃 Happy Face",
+              url: "https://i.imgur.com/6Jcfgnp.jpg",
+              caption: "This is a happy face"
+            },
+            {
+              title: "😮",
+              url: "https://i.imgur.com/OsLxYkT.jpg",
+              caption: "Suprised face"
+            },
+            {
+              url: "https://i.imgur.com/uigfzs5.jpg"
+            }
+          ]}
+          onRequestClose={() => isOpen(false)}
+        />
+      </>
+    );
+  }}
+</Playground>
+
+## Design & Usage Guidelines
+
+A lightbox should:
+
+- Be used when viewing an image or collection of images is the user's primary
+  goal. This shouldn't be used for things like in product upgrade prompts.
+- Resemble the industry standard lightbox, using Jobber's font and color
+  palette. Novel lightbox functionality will surprise and confuse users.
+
+## Props
+
+<Props of={LightBox} />
+
+---
+
+## open
+
+Specify if the Lightbox is open or closed.
+
+## Images
+
+Images is an array of objects defining a LightBox image. This object consists of
+`title`, `caption` and `url`. `title` and `caption` are optional, `url` is
+required, for each image.
+
+## Image Index
+
+Use this to specify which image in `images` to initialize the lightbox with.
+This is useful when you have a collection of thumbnails as you only need one
+collection of image urls, order doesn't matter.
+
+## onRequestClose
+
+This function must set open to false in order to close the lightbox. Note there
+is a 300ms easing animation on lightbox close that occurs before this function
+is called.
```

Comment:
These comments should be moved in with the props in the `tsx` file. Then they will properly show up in the props table.

eg: https://github.com/GetJobber/atlantis/blob/master/packages/components/src/Avatar/Avatar.tsx#L9-L31

![image](https://user-images.githubusercontent.com/131327/81750264-65961400-946a-11ea-8e97-b024f9d2ef8b.png)